src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -538,13 +538,12 @@ def check_delete_job(configuration, namespace):
         logger.debug("Error occurred while retrieving status of the delete job: {}".format(str(e)))
 
 
-def try_upload_log_file(cluster_name, storage_account_name, storage_token, log_file_path):
+def try_upload_log_file(storage_account_name, storage_token, container_name, log_file_path):
```

Comment:
Lets not make container name a required parameter.

Probably keep some default name as well, like "connectedk8s_troubleshoot_logs" or something